spec/plans/guest-topology.fmf

```diff
@@ -0,0 +1,114 @@
+summary: Define format of on-disk description of provisioned guest topology
+title: Guest topology format
```

Comment:
```suggestion
title: Guest Topology Format
```

Consistent with other headings in the section.

Reply:
Fixed in 2526d401.